schemas/schema-schema.ipldsch

```diff
@@ -148,30 +148,131 @@ type AnyScalar union {
 	| Float float
 } representation kinded
 
+## AdvancedDataLayout holds `advanced` definitions which currently only require
+## a name.
+##
+## Use `advanced Foo` rather than `type Foo` to indicate an advanced data layout
+## (ADL) with that name which can be used as a representation for type
+## definitions whose kind the ADL is able to support.
+##
+## The ADL name is currently the only identifier that can be used to make a
+## connection with the algorithm/logic behind this ADL. Future iterations may
+## formalize this connection by some other means.
+##
+type AdvancedDataLayout struct {
+	name String
```

Comment:
This field might disappear entirely if the [two maps proposal](https://github.com/ipld/specs/pull/182#issuecomment-531554056) makes sense.  Or if not, would become `AdvancedLayoutRef` per https://github.com/ipld/specs/pull/182/files#r324478807 .